rust/benchmarks/src/bin/tpch.rs

```diff
@@ -353,3 +353,36 @@ fn get_schema(table: &str) -> Schema {
         _ => unimplemented!(),
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::env;
+
+    #[tokio::test]
+    async fn q1() -> Result<()> {
+        verify_query(1).await
+    }
+
+    #[tokio::test]
+    async fn q12() -> Result<()> {
+        verify_query(12).await
+    }
+
+    async fn verify_query(n: usize) -> Result<()> {
```

Comment:
maybe also test the mem_table / some other options by adding at as parameter in `verify_query` ?